distribution/test/itests/README.md

```diff
@@ -16,6 +16,16 @@ In short:
 * Use remote debugging to debug tests themselves or anything related to production code
 * Use local debugging to debug the code that sets up and configures the testing environment
 
+## Solr
+Solr runs in its own OS process separate PAX-EXAM and the application. Things to know include:
+
+* The default port for Solr is hardcoded to `9994`. 
+* The Solr administration UI is available at `http://localhost:9994/solr`
+* The Maven exec-maven-plugin executes the Solr start command before the itests begin. The plugin also executes the Solr stop command when the itests have terminated.
+* The files for the Solr instance are located inside of the `test-itest-ddf/target/solr` folder and **not** in the exam folder. There is not a per exam-folder copy of Solr. Therefore it is not
+ recommended to run itests concurrently because the tests may corrupt each other's data.
+* Because not every itest cleans up after itself, it is possible to leak data between separate runs of the itests, **if** the test process terminated abnormally. In such cases, use `mvn clean` to delete the target folder. This ensures future itests have a clean Solr installation.
```

Comment:
We could tweak the itest setup to sanitize the solr catalog at the beginning of a run.

Reply:
I think @ahoffer has already generated a follow on ticket to track this edge case.